composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "arlo/moodle-enrol_arlo",
     "description": "Arlo integration for Moodle",
     "require": {
-        "php": ">=7.0 <7.3",
+        "php": ">=7.0",
```

Comment:
Have you tested this plugin in 7.4+? Any deprecated  code?

Reply:
Yes, I have tested it locally on 7.4. All tests were passing

Reply:
good. Can you please also 
1. Create an issue in the upstream repo
2. Create a pull request with this change.
Thanks!